tools/scxtop/src/app.rs

```diff
@@ -2073,10 +2073,10 @@ impl<'a> App<'a> {
             Action::PageDown => self.on_pg_down(),
             Action::Enter => self.on_enter(),
             Action::SetState(state) => {
-                if state == self.state {
+                if *state == self.state {
```

Comment:
For now it's just pushing the clones further down, but this can be cleaned up later.